### PR TITLE
WIP: using istioctl to convert Ingress

### PR DESCRIPTION
**Pull request by esnible**

### CODE REVIEW

File: content/blog/2018/migrating-gateway/index.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+---
+title: Migrating to the Istio Gateway
+subtitle: Replace your Ingresses with Gateways using Istio 1.0
+description: Converting and replacing your Kubernetes Ingresses with Istio Gateways
+publishdate: 2018-08-15
+attribution: The Istio Team
```
Comment (linsun) [inline content/blog/2018/migrating-gateway/index.md]: would not this be you?  I think the attribution is istio team for release announcement.

File: content/blog/2018/migrating-gateway/index.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+not work unless you configured Istio to include Ingress support.  We don't need to do that
+because we will immediately convert this Ingress to the new format.
+
+## Deploying your Gateway
+
+The `istioctl` CLI includes a command to convert an Ingress.  Let's perform the conversion.
+Replace _simple-ingress_ with the name of your Ingress.
+
+```
+kubectl get ingress simple-ingress -o yaml > ingress.yaml
```
[the remark this answers is not in the document]
Comment (linsun) [reply content/blog/2018/migrating-gateway/index.md]: I think the auto convert isn't enabled by default.... you'd have to enable the old istio ingress to enable it and set global.k8sIngressSelector=ingressgateway via helm based install.

**Comment by geeknoid** [comment]
Is this PR still needed, or should we close it?

**Comment by geeknoid** [comment]
Closing due to inactivity.

**Pull request closed without merging**